importers/replay_contact_importer.py

```diff
@@ -14,12 +14,11 @@
 
 
 class ReplayContactImporter(Importer):
-    def __init__(
-        self,
-        name="Replay Contact Importer",
-        validation_level=constants.ENHANCED_LEVEL,
-        short_name="REP Contact Importer",
-    ):
+    def __init__(self):
+        name = "Replay Contact Importer"
+        validation_level = constants.ENHANCED_LEVEL
+        short_name = "REP Contact Importer"
+
```

Comment:
@robintw - aren't we still creating some objects unnecessarily?

Can't we use: 
```
super().__init__(
        name = "Replay Contact Importer"
        validation_level = constants.ENHANCED_LEVEL
        short_name = "REP Contact Importer"
)
```

We may require the `short_name` later on, for error reporting, but I guess the parent-level version is available .

Reply:
Yes of course we can - I was obviously being dumb yesterday. Fixed now.